src/components/Modal/helper.js

```diff
@@ -24,3 +29,26 @@ export const isRequestCloseApproved = ({ onRequestClose, event }) => {
     .then(result => (typeof result === "undefined" ? true : Boolean(result)))
     .catch(() => false);
 };
+
+export const throttle = (callback, wait, immediate = false) => {
```

Comment:
Please relocate this function to `utils`

Reply:
Sure